applications/incompressible_navier_stokes/free_stream/tests/coupled_bdf2_moving_mesh.output

```diff
@@ -196,74 +196,62 @@ ________________________________________________________________________________
  Time step number = 1       t = 0.00000e+00 -> t + dt = 8.50500e-03
 ________________________________________________________________________________
 
-Solve linear problem:
-  Iterations:              0
-
 Calculate error for velocity at time t = 1.0004e+00:
-  Relative error (L2-norm): 1.25049e-15
+  Relative error (L2-norm): 9.03070e-16
```

Comment:
Why did the values change here?

Reply:
I think this is something that we have to live with. However, this is no problem since this is covered by numdiff.